src/pages/createPledge.js

```diff
@@ -222,7 +222,7 @@ class CreatePledgePage extends React.Component {
     if (data.Collective) {
       website = data.Collective.githubHandle
         ? `https://github.com/${data.Collective.githubHandle}`
-        : data.Collective.website;
```

Comment:
Same here too....a mistake too..I actually thought I was doing all of these on the other copy on the file I created I didn't know it was the main file am supposed to push...will retrace it by tomorrow